tests/app/forms/test_field_factory.py

```diff
@@ -1,68 +1,66 @@
-from app.data_models.answer_store import AnswerStore
-from app.data_models.list_store import ListStore
+import pytest
+
 from app.forms import error_messages
 from app.forms.field_handlers import get_field_handler
 from app.questionnaire import QuestionnaireSchema
 from app.questionnaire.rules.rule_evaluator import RuleEvaluator
 from app.questionnaire.value_source_resolver import ValueSourceResolver
-from tests.app.app_context_test_case import AppContextTestCase
 
 
-class TestFieldFactory(AppContextTestCase):
-    def test_invalid_field_type_raises_on_invalid(self):
-        schema = QuestionnaireSchema(
-            {
-                "questionnaire_flow": {
-                    "type": "Linear",
-                    "options": {"summary": {"collapsible": False}},
-                }
+def test_invalid_field_type_raises_on_invalid(answer_store, list_store):
+    schema = QuestionnaireSchema(
+        {
+            "questionnaire_flow": {
+                "type": "Linear",
+                "options": {"summary": {"collapsible": False}},
             }
-        )
-
-        metadata = {
-            "user_id": "789473423",
-            "schema_name": "0000",
-            "collection_exercise_sid": "test-sid",
-            "period_id": "2016-02-01",
-            "period_str": "2016-01-01",
-            "ref_p_start_date": "2016-02-02",
-            "ref_p_end_date": "2016-03-03",
-            "ru_ref": "432423423423",
-            "ru_name": "Apple",
-            "return_by": "2016-07-07",
-            "case_id": "1234567890",
-            "case_ref": "1000000000000001",
         }
+    )
 
-        response_metadata = {}
+    metadata = {
```

Comment:
We should probably make `metadata` an empty dict similar to `response metadata` as it's not needed for the test and passes without any key/value pairs in it (not required  for `ValueSourceResolver` and  `RuleEvaluator` to work).